FE/src/components/UserInfo.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function UserInfo ({token, adminPriv, userData}) {
+    // console.log(userData);
+    // console.log(adminPriv);
+    return (
+        <div className = 'userInfo'>
+            <img src={userData.imgurl} alt={`User account image for ${userData.username}`} height="15%" width="22.5%"/>
```

Comment:
I agree with Jared's comment, but am commenting again to remove the width and height attributes from these images.